src/main/scala/com/yotpo/metorikku/input/kafka/KafkaInput.scala

```diff
@@ -16,4 +24,13 @@ case class KafkaInput(name: String, servers: Seq[String], topic: String,
     }
     inputStream.load()
   }
-}
+
+  private def createKafkaConsumer = {
+    val props = new Properties()
+    props.put("bootstrap.servers", servers.mkString(","))
+    props.put("group.id", consumerGroup)
+    props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer")
```

Comment:
Not sure you need these... Try without

Reply:
We need them, otherwise it throws an exception that we need Serializers